developers/ide/eclipse.md

```diff
@@ -5,65 +5,170 @@ title: Eclipse IDE
-Ensure that you select the Eclipse version to use to "2019-03" as version `2019-06` currently does not support bndtools. Finally select `Next >`.
+1. Launch the Installer and on the menu on the top right (3 bars)
 
-1. Under `Github Projects > openHAB` select `openHAB Development` and any desired option from `openHAB Add-ons` (includes all add-ons from openhab2-addons repo), `openHAB ZigBee Binding` or `openHAB Z-Wave Binding`.
+    ![installer](./images/ide_setup_eclipse_1_installer.png)
 
-   ![select projects](./images/ide_setup_eclipse_projects.png)
+1. Select `ADVANCED MODE...`
 
-1. Click `Next>`, verify/modify Root and install folder name, click `Next>` and `Finish` to start installation. 
-    During install accept licence agreement, "Unsigned Content" for Bndtools, and Eclipse Foundation certificates when requested to complete IDE installation. 
-    When the Eclipse installation is finished the IDE is launched automatically.
+    ![installer](./images/ide_setup_eclipse_2_advanced.png)
 
-1. **It's important**, during the first Eclipse IDE launch, to leave Eclipse open until all openHAB related initial Setup tasks / Download sources / Builds are completed. 
+1. Select `Eclipse IDE for Java Developers`. Select `Next >`.
 
-    Setup tasks will personalize the IDE with openHAB code formatting tools, configurations and a demo app.
-	Setup tasks will also download openHAB lastest `openhab2-addons` and `openhab-distro` code repositories and related maven dependencies. 
-	Click bottom right button in the IDE for Progress.
+    ![select](./images/ide_setup_eclipse_3_select_ide.png)
+
+1. Under `GitHub Projects > openHAB` select `openHAB Development` and any desired option from `openHAB Add-ons` (includes all add-ons from openhab2-addons repo), `openHAB ZigBee Binding` or `openHAB Z-Wave Binding`.
+
+   ![select projects](./images/ide_setup_eclipse_4_openhab.png)
 
-    ![startup tasks progress](./images/ide_setup_eclipse_startup_tasks.png)
+    | Selection               | Install if                            |
+    |-------------------------|---------------------------------------|
+    | **openHAB Development** | **Debug/Demo Environment (Required)** |
+    | openHAB Add-ons         | Installs all openHAB 2 Add-ons        |
+    | openHAB ZigBee Binding  | ZigBee Binding Development            |
+    | openHAB Z-Wave Binding  | Z-Wave Binding Development            |
+    | openHAB BACNet Binding  | BACNet Binding Development            |
+    | openHAB Web UIs         | Web UIs Development                   |
+    | openHAB Core Framework  | Core Framework Development            |
 
-    When visible in the Package Explorer, select `OH2 Add-ons` and from right-click menu select "Close Projects": this significantly speeds up the setup.
+    ::: warning Attention
+    If you have selected `openHAB Add-ons` the installer will check out the openHAB2-addons repository  and all add-on projects are imported in Eclipse.
```

Comment:
Just as a cross-link. You can apply the suggestion right away if you agree on it.

```suggestion
    If you have selected `openHAB Add-ons` the installer will check out the [openHAB 2.x Add-ons](https://github.com/openhab/openhab2-addons/) repository and all add-on projects are imported in Eclipse.
```